### PR TITLE
fix(OnyxSelect): support skeleton and textMode property

### PR DESCRIPTION
**Pull request by larsrickert**
Relates to #565, #706

- fix skeleton and textMode prop not working
- update Storybook examples with readonly, disable, skeleton and multiselect text mode
- update screenshot tests to cover ore use cases
- remove no longer needed loading indicator inside the flyout since the loading is only shown in the `OnyxSelectInput` and the flyout can not be opened then

## Checklist

- [x] If a new component is added, at least one [Playwright screenshot test](https://github.com/SchwarzIT/onyx/actions/workflows/playwright-screenshots.yml) is added
- [x] A changeset is added with `npx changeset add` if your changes should be released as npm package (because they affect the library usage)

### CODE REVIEW

File: packages/sit-onyx/playwright/snapshots/components/OnyxSelect/Select-multiple--chromium-linux.png
Comment (BoppLi) [inline packages/sit-onyx/playwright/snapshots/components/OnyxSelect/Select-multiple--chromium-linux.png:1]: the top row is named "empty", there I would expect the "void" component in the flyout given that name? or what does the name actually mean

Comment (larsrickert) [reply packages/sit-onyx/playwright/snapshots/components/OnyxSelect/Select-multiple--chromium-linux.png:1]: That was a leftover, I renamed it

File: packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.stories.ts [hunk 1]
```diff
@@ -222,13 +233,42 @@ export const ButtonLoading = {
...
+ * This example shows a disabled select that can not be edited and is therefore not included in any form data.
+ */
+export const Disabled = {
+  args: {
+    ...Default.args,
+    disabled: true,
+  },
+} satisfies Story;
+
+export const Skeleton = {
```
[the remark this answers is not in the document]
Comment (larsrickert) [reply packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.stories.ts:263]: Done

**Review by BoppLi — APPROVED** [review]

File: packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.stories.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (larsrickert) [reply packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.stories.ts:263]: Imho "This example shows a skeleton select" is kind if useless :D I know we already have such docs for other comments but I'd prefer to reduce this

[the remark this answers is not in the document]
Comment (BoppLi) [reply packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.stories.ts:263]: the we might want to remove this on all the other places as well? should we create a cleanup ticket?

**Pull request merged**